BingusApi/faq_config.json

```
@@ -665,6 +665,26 @@
         "whats the new location for config and logs now?",
         "where did the configs go"
       ]
+    },
+	{
+      "title": "Official Trackers - Blue Light of Death (BLOD)",
+      "answer": "Symptoms of this hardware issue that may occur on some of the **official** trackers include solid blue light (not blinking), suddenly stopped working, or suddenly stopped connecting to WiFi/SlimeVR server. Direct message Eiren with your order number and shipping details for instructions of getting your replacement tracker(s).",
+      "matched_questions": [
+        "solid blue light",
+        "tracker stopped working",
+        "tracker stopped connecting",
+        "blod"
+      ]
+    },
+	{
```

Inconsistent spaces and tabs (it should be 2 spaces indentation in this file), please fix the indentation on these opening curly brackets.
```suggestion
    {
      "title": "Official Trackers - Blue Light of Death (BLOD)",
      "answer": "Symptoms of this hardware issue that may occur on some of the **official** trackers include solid blue light (not blinking), suddenly stopped working, or suddenly stopped connecting to WiFi/SlimeVR server. Direct message Eiren with your order number and shipping details for instructions of getting your replacement tracker(s).",
      "matched_questions": [
        "solid blue light",
        "tracker stopped working",
        "tracker stopped connecting",
        "blod"
      ]
    },
    {
```